### PR TITLE
Globalenv fix for CRAN submission

### LINKED ISSUE #194: Add release checklist to README
Compile/Update:
1. README.Rmd
2. vignette
3. pkgdown page
4. NEWS.md

### PR DESCRIPTION
**Pull request by aghaynes**
closes #194

### CODE REVIEW

**Comment by aghaynes** [comment]
@PatrickRWright can you build the pkgdown, readme and vignette? Something seems odd on mine (seems to be a lot of changes, particularly in the readme. pkgdown might be OK though)

**Comment by PatrickRWright** [comment]
Thanks @aghaynes 

I'll do that, yes. I'll also add the checklist to the README (i.e. #194)

File: README.Rmd [hunk 1]
```diff
@@ -190,7 +190,7 @@ Working with a list can be tiresome so `secuTrialR` provides a `as.data.frame` m
 ```{r}
 ls()
 names(ctu05)
-as.data.frame(ctu05)
+as.data.frame(ctu05, envir = .GlobalEnv)
```
Comment (PatrickRWright) [inline README.Rmd:193]: Not sure if this is still in violation with CRAN regulations...

File: README.md [hunk 2]
```diff
@@ -320,24 +322,35 @@ names(ctu05)
 ```
 
 ```r
-as.data.frame(ctu05)
+as.data.frame(ctu05, envir = .GlobalEnv)
```
Comment (PatrickRWright) [inline README.md:325]: same here.

File: R/asdataframe.R [hunk 3]
```diff
@@ -18,16 +24,20 @@
...
-#' # add files to global environment, removing the project name from the file names
-#' as.data.frame(sT_export, regex = "ctu05")
-as.data.frame.secuTrialdata <- function(x, ...,
+#' # add files to a new environment called env1
+#' env1 <- new.env()
+#' as.data.frame(sT_export, envir = env1)
+#' # add files to a new environment called env2, removing the project name from
+#' # the file names
+#' env2 <- new.env()
+#' as.data.frame(sT_export, regex = "ctu05", envir = env)
```
Comment (PatrickRWright) [inline R/asdataframe.R]: env2 ?

**Comment by aghaynes** [comment]
thanks :)

File: README.Rmd (hunk 1 above)
Comment (aghaynes) [reply README.Rmd]: thanks, I missed that one

File: README.md (hunk 2 above)
Comment (aghaynes) [reply README.md]: (generated from the Rmd ;) )

**Pull request merged**